comm/src/main/scala/coop/rchain/comm/transport/TcpTransportLayer.scala

```diff
@@ -67,15 +60,42 @@ class TcpTransportLayer[F[_]: Monad: Capture: Metrics: Futurable](
       .overrideAuthority(remote.id.toString)
       .build()
 
+  private def transportLayerStub(remote: PeerNode): F[TransportLayerStub] =
```

Comment:
rename to get connection? `transportLayerStub` seems really like name with zero value. but i'm not certain, convince me wrong please :)

Reply:
ConnectionsState hold Connections which are named stubs. This is confusing.